app/components/forms/wizard/other-details-step.js

```diff
@@ -0,0 +1,563 @@
+            },
+            {
+              type   : 'integer[1..]',
+              prompt : this.l10n.t('Minimum price needs to be greater than zero')
+            },
+            {
+              type   : 'checkMaxMinPrice',
+              prompt : this.l10n.t('Minimum price should not be greater than maximum')
+            }
+          ]
+        },
+        maxPrice: {
+          identifier : 'max_price',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.l10n.t('Maximum price for donation tickets required')
+            },
+            {
+              type   : 'integer[1..]',
+              prompt : this.l10n.t('Maximum price needs to be greater than zero')
+            },
+            {
+              type   : 'checkMaxMinPrice',
+              prompt : this.l10n.t('Maximum price should not be less than minimum')
+            }
+          ]
+        },
+        paypalEmail: {
+          identifier : 'paypal_email',
+          rules      : [
+            {
+              type   : 'email',
+              prompt : this.l10n.t('Please enter a valid email')
+            },
+            {
+              type   : 'empty',
+              prompt : this.l10n.t('Please fill your paypal email for payment of tickets.')
+            }
+          ]
+        },
+        onSiteDetails: {
+          identifier : 'on_site_details',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.l10n.t('Please fill the details for payment of tickets.')
+            }
+          ]
+        },
+        bankDetails: {
+          identifier : 'bank_details',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.l10n.t('Please fill the bank details for payment of tickets.')
+            }
+          ]
+        },
+        chequeDetails: {
+          identifier : 'cheque_details',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.l10n.t('Please fill the cheque details for payment of tickets.')
+            }
+          ]
+        },
+        externalEventIdentifier: {
+          identifier : 'external_event_url',
+          optional   : true,
+          rules      : [
+            {
+              type   : 'regExp',
+              value  : protocolLessValidUrlPattern,
+              prompt : this.l10n.t('Please enter a valid url')
+            }
+          ]
+        },
+        paymentCountry: {
+          identifier : 'payment_country',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.l10n.t('Please select your country')
+            }
+          ]
+        }
+      }
+    };
+    // Merging the predetermined rules with the rules for social links.
+    validationRules.fields = merge(validationRules.fields, this.socialLinksValidationRules);
+    return validationRules;
+  },
+
+  actions: {
+    
```

Comment:
Codacy found an issue: [Trailing spaces not allowed.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6089057)